command/harness/dlite/dlite.go

```diff
@@ -24,19 +24,20 @@ import (
 )
 
 type dliteCommand struct {
-	envFile         string
-	env             config.EnvConfig
-	delegateInfo    *poller.DelegateInfo
-	poolFile        string
-	poolManager     *drivers.Manager
-	metrics         *metric.Metrics
-	stageOwnerStore store.StageOwnerStore
+	envFile                string
+	env                    config.EnvConfig
+	delegateInfo           *poller.DelegateInfo
+	poolFile               string
+	poolManager            drivers.IManager
+	distributedPoolManager drivers.IManager
+	metrics                *metric.Metrics
 }
 
 func RegisterDlite(app *kingpin.Application) {
 	c := new(dliteCommand)
 
 	c.poolManager = &drivers.Manager{}
```

Comment:
Can we set the pool manager just once? That would avoid having `poolManager` and `distributedPoolManager` everywhere even though only one will ever be used. If you need to check which case it is, you could do so from the env (`if SHARED_POOL...`)